repository/student/student.go

```diff
@@ -0,0 +1,89 @@
+// Package student provides the data structures and repository methods
+// for managing student records.
+package student
+
+import (
+	"context"
+	"database/sql"
+	"log"
+	"time"
+)
+
+// Student represents a student record in the database.
+type Student struct {
+	ID        string
+	FirstName string
+	LastName  string
+	Grade     int32
+	CreatedAt time.Time
+}
+
+// Repository manages operations with students in the DB.
+type Repository struct {
+	db *sql.DB
+}
+
+// NewRepository creates a new Repository with the given DB connection.
+func NewRepository(db *sql.DB) *Repository {
+	return &Repository{db: db}
+}
+
+// Create inserts a new student record into the database.
+func (r *Repository) Create(ctx context.Context, s Student) (string, error) {
+	_, err := r.db.ExecContext(ctx,
+		`INSERT INTO students (id, first_name, last_name, grade, created_at)
+		 VALUES ($1, $2, $3, $4, $5)`,
+		s.ID, s.FirstName, s.LastName, s.Grade, s.CreatedAt,
```

Comment:
Lets use sqlx package to avoid writing each field every time and use struct tags to manage mapping of fields to database columns
Also, lets have an array of all columns stored once, smth like 
```
var columns = [...]string{"id", ...}
``` 

and a function to make it a string like `id, first_name, last_name, grade, created_at`, that we need in queries